src/components/track-tile/TrackTileArt.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { memo } from 'react'
+
+import {
+  useTrackCoverArt,
+  useCollectionCoverArt,
+  useLoadImageWithTimeout
+} from 'audius-client/src/common/hooks/useImageSize'
+import { ID } from 'audius-client/src/common/models/Identifiers'
+import {
+  CoverArtSizes,
+  SquareSizes
+} from 'audius-client/src/common/models/ImageSizes'
+import { Remix } from 'audius-client/src/common/models/Track'
+import { StyleProp, StyleSheet, View, ViewStyle } from 'react-native'
+
+import CoSign, { Size } from 'app/components/co-sign'
+import DynamicImage from 'app/components/dynamic-image'
+
+type TrackTileArtProps = {
+  isTrack: boolean
+  id: ID
+  coverArtSizes: CoverArtSizes
+  style?: StyleProp<ViewStyle>
+  showSkeleton?: boolean
+  coSign?: Remix | null
+  onLoad: () => void
+}
+
+const styles = StyleSheet.create({
+  image: {
+    borderRadius: 4,
+    height: 74,
+    width: 74
+  }
+})
+
+const TrackTileArt = ({
+  id,
+  isTrack,
+  style,
+  coverArtSizes,
+  showSkeleton,
+  coSign,
+  onLoad
+}: TrackTileArtProps) => {
+  const useImage = isTrack ? useTrackCoverArt : useCollectionCoverArt
+  const image = useImage(id, coverArtSizes, SquareSizes.SIZE_150_BY_150)
+
+  useLoadImageWithTimeout(image, onLoad)
+
+  const imageElement = (
+    <DynamicImage
+      image={showSkeleton ? undefined : { uri: image }}
+      style={styles.image}
+    />
+  )
+
+  return coSign ? (
+    <CoSign
+      size={Size.SMALL}
+      style={[style, styles.image]}
+      hasFavorited={coSign.has_remix_author_saved}
+      hasReposted={coSign.has_remix_author_reposted}
+      coSignName={coSign.user.name}
+      userId={coSign.user.user_id}
+    >
+      {imageElement}
+    </CoSign>
+  ) : (
+    <View style={[style, styles.image]}>{imageElement}</View>
+  )
+}
+
+export default memo(TrackTileArt)
```

Comment:
something ive been seeing a bit, where we default export a memoized comp. from what i can tell we should only use `memo` for places that can truly benefit from perf optimization, and using it can lead to bugs, so just curious of your thoughts there! the other thing that personally makes dev process harder, is the default export results in a component with an undefined name, so in the component heirarchy i see lots of layers of anonymous component names.

Reply:
I agree that the `memo` is overkill in most places, I copied this one over from the existing TrackTileArt in the web client. @raymondjacobson would probably be the person to discuss advantages/disadvantages of using memoized components with!

I'm 100% on board with banning default exports, I already have a branch adding that eslint rule. I think the rest of the team is on board too but we can discuss again. Maybe we start doing only named exports moving forward and then we do a sweep of the mobile app and replace all existing default exports